wandb/sdk/launch/_project_spec.py

```diff
@@ -153,6 +156,10 @@ def add_entry_point(self, entry_point: str) -> "EntryPoint":
             )
         )
 
+    def _update_base_image_uid(self) -> None:
```

Comment:
this function updates the current image uid to the base image uid right? could you rename to something like `_update_uid_to_base_image_uid` (verbose ugh but idk), the current fn name sounds like it's doing the opposite thing

Reply:
Yeah, you're right.